pom.xml

```diff
@@ -19,6 +19,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
```

Comment:
please use latest version of Java (14) or at least latest LTS version (11) also take a look at latest language festures and try to use them in practice